downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -42,9 +40,9 @@ public static void processMolecularFunctions(GKInstance reactionInst) throws Exc
      * Then it will check if the ActiveUnit is valid if it exists. If there is no ActiveUnit, the PhysicalEntity
      * is checked to see if it is 'multi-instance', which will invalidate it.
      * Catalysts with more than 1 ActiveUnit are also considered invalid.
-     * @param catalystInst -- Catalyst instance taken from the original reaction instance.
-     * @return
-     * @throws Exception
+     * @param catalystInst -- GKInstance, catalyst instance taken from the original reaction instance.
+     * @return -- True if the catalyst instance has been deemed valid, false if not.
```

Comment:
The type of the return value (i.e. boolean) could be included here for consistency with the other methods.